docs/process.md

```diff
@@ -1057,43 +1057,38 @@ To sum up, the use of output files with static names over dynamic ones is prefer
 
 The `env` qualifier allows you to output a variable defined in the process execution environment:
 
-```groovy
-process myTask {
-    output:
-    env FOO
-
-    script:
-    '''
-    FOO=$(ls -la)
-    '''
-}
-
-workflow {
-    myTask | view { "directory contents: $it" }
-}
+```{literalinclude} snippets/process-out-env.nf
+:language: groovy
 ```
 
+If the environment variable contains multiple lines of output, the output is compressed to a single line by converting newlines to spaces.
+
 (process-stdout)=
 
 ### Output type `stdout`
 
 The `stdout` qualifier allows you to output the `stdout` of the executed process:
 
-```groovy
-process sayHello {
-    output:
-    stdout
+```{literalinclude} snippets/process-stdout.nf
+:language: groovy
+```
 
-    """
-    echo Hello world!
-    """
-}
+(process-out-cmd)=
 
-workflow {
-    sayHello | view { "I say... $it" }
-}
+### Output type `cmd`
+
+The `cmd` qualifier allows you to capture the standard output of an arbitrary shell command:
+
+```{literalinclude} snippets/process-out-cmd.nf
+:language: groovy
 ```
 
+Only one-line Bash commands are supported. You can use a semi-colon `;` to specify multiple Bash commands on a single line, and many interpreters can execute arbitrary code on the command line, e.g. `python -c 'print("Hello world!")'`.
+
+If the command fails, the task will also fail. In Bash, you can append `|| true` to a command to suppress any command failure.
```

Comment:
> To make it fails on the very first error it may be enough pretending the command with set -e; (or pipeline?)

Unfortunately no, even with `bash -ueo pipefail` it still doesn't fail. I can't find a way to make it fail without assigning it to an environment var

Reply:
As a workaround we could do:
```bash
FOO=$(bash --version) # fails if inner command fails
echo FOO="$FOO"
```

Reply:
resolved